components/vf-navigation/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.2.3-3.0.0-beta.0.0",
+  "version": "3.0.0-beta.0",
```

Comment:
should this be `beta.1` to match the changelog?

Reply:
ah, lerna will bump it to .1